robot/control/main.cpp

```diff
@@ -145,51 +196,16 @@ int main() {
     moduleList.emplace_back(curTime, KickerModule::period,        KickerModule::runtime,        &kicker);
     moduleList.emplace_back(curTime, BatteryModule::period,       BatteryModule::runtime,       &battery);
     moduleList.emplace_back(curTime, RotaryDialModule::period,    RotaryDialModule::runtime,    &dial);
-    moduleList.emplace_back(curTime, LEDModule::period,           LEDModule::runtime,           &led);
+    moduleList.emplace_back(curTime, LEDModule::period,           LEDModule::runtime,           led.get());
 
-    while (true) {
-        // Must do all timing in systick
-        // Base time we use is in systick, and if you convert from systick
-        // to us and try to compare timepoints, the value will "overflow" at
-        // some random point which messes up the `Current - Start` magic
-        // that automatically deals with delta time using the math behind overflows
-        uint32_t loopStartTime = DWT_GetTick();
-        uint32_t loopEndTime = DWT_GetTick() + SUPER_LOOP_PERIOD*DWT_SysTick_To_us();
+    TaskHandle_t superLoopHandle;
+    xTaskCreate(superLoopTask, "Super Loop", 1 << 15, nullptr, 1, &superLoopHandle);
```

Comment:
`size_t` is larger than `1 << 15` right?

Reply:
Yeah, size_t should be 32 bits.